sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,372 @@
+}
+
+// @public
+export type DigitalTwinsListRelationshipsResponse = RelationshipCollection & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: RelationshipCollection;
+    };
+};
+
+// @public
+export interface DigitalTwinsSendComponentTelemetryOptionalParams extends coreHttp.RequestOptionsBase {
+    dtTimestamp?: string;
+}
+
+// @public
+export interface DigitalTwinsSendTelemetryOptionalParams extends coreHttp.RequestOptionsBase {
+    dtTimestamp?: string;
+}
+
+// @public
+export interface DigitalTwinsUpdateComponentHeaders {
+    eTag: string;
+}
+
+// @public
+export interface DigitalTwinsUpdateComponentOptionalParams extends coreHttp.RequestOptionsBase {
+    ifMatch?: string;
+    patchDocument?: any[];
+}
+
+// @public
+export type DigitalTwinsUpdateComponentResponse = DigitalTwinsUpdateComponentHeaders & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: DigitalTwinsUpdateComponentHeaders;
+    };
+};
+
+// @public
+export interface DigitalTwinsUpdateHeaders {
+    eTag: string;
+}
+
+// @public
+export interface DigitalTwinsUpdateOptionalParams extends coreHttp.RequestOptionsBase {
+    ifMatch?: string;
+}
+
+// @public
+export interface DigitalTwinsUpdateRelationshipHeaders {
+    eTag: string;
+}
+
+// @public
+export interface DigitalTwinsUpdateRelationshipOptionalParams extends coreHttp.RequestOptionsBase {
+    ifMatch?: string;
+    patchDocument?: any[];
+}
+
+// @public
+export type DigitalTwinsUpdateRelationshipResponse = DigitalTwinsUpdateRelationshipHeaders & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: DigitalTwinsUpdateRelationshipHeaders;
+    };
+};
+
+// @public
+export type DigitalTwinsUpdateResponse = DigitalTwinsUpdateHeaders & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: DigitalTwinsUpdateHeaders;
+    };
+};
+
+// @public
+export interface EventRoute {
+    endpointName: string;
+    filter?: string;
+    readonly id?: string;
+}
+
+// @public
+export interface EventRouteCollection extends Array<EventRoute> {
+    nextLink?: string;
+}
+
+// @public
+export interface EventRoutesAddOptionalParams extends coreHttp.RequestOptionsBase {
+    eventRoute?: EventRoute;
+}
+
+// @public
+export type EventRoutesGetByIdResponse = EventRoute & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: EventRoute;
+    };
+};
+
+// @public
+export type EventRoutesListNextResponse = EventRouteCollection & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: EventRouteCollection;
+    };
+};
+
+// @public
+export interface EventRoutesListOptionalParams extends coreHttp.RequestOptionsBase {
+    eventRoutesListOptions?: EventRoutesListOptions;
+}
+
+// @public
+export interface EventRoutesListOptions {
+    maxItemCount?: number;
+}
+
+// @public
+export type IfNoneMatch = '*';
+
+// @public
+export type IfNoneMatch1 = '*';
```

Comment:
Duplicate declaration here.

Reply:
Added to the swagger changes list

---
In reply to: [455325165](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r455325165) [](ancestors = 455325165)